electron/launcher.ts

```diff
@@ -96,7 +96,8 @@ async function launch(
       return
     }
   }
-  const exe = targetExe && isLegendary ? `--override-exe ${targetExe}` : ''
+  const exe =
```

Comment:
is this possible already with `gogdl`, @imLinguin ?
There is any risk of breaking GOGDL or we can keep it?

Reply:
It's possible:
https://github.com/Heroic-Games-Launcher/heroic-gogdl/commit/bf0ab9d5b39734e9f364a5b947f5f03b29f45308
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/commit/d29ac86a64fc1335d1c49a5d2422aa7ce55dc510

It's unlikely that it will break anything.
I tested it on two games with and without custom exe.
It wasn't previously supported.
But now that I think of it, it's possible that user have set custom exe before, it didn't work so they ignored it and now after an update they will be surprised when game doesn't launch but run some random exe they have set earlier.

Reply:
Yes it's completely ready on `gogdl` side, I forgot to change that if statement :shrug: 

As a sidenote: It'll use the provided exe's directory as working dir

Reply:
> As a sidenote: It'll use the provided exe's directory as working dir
So there shouldn't be a reason for heroic spawn to also be in that directory

changes for setting working directory is not for alternative exe to run, but run exe on prefix